File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { StarWarsDatabankSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await StarWarsDatabankSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- StarWarsDatabank SDK exists test

local sdk = require("star-wars-databank_sdk")

describe("StarWarsDatabankSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
